internal/web/playground.css

```diff
@@ -1,17 +1,66 @@
 :root {
     --main-font-family: -apple-system, BlinkMacSystemFont, Helvetica, Arial, sans-serif;
+    
+    --white: #fff;
+
+    --toolbar-background: #353535;
+    --toolbar-color: #d3d3d3;
+    --ctr-input-border: #808080;
+    --ctr-input-background: #ececec;
+    --btn: #375eab;
+    --btn-hover: #1f3663;
+    
+    --h3-color: #24292e;
+    --window-border: #e5e7e8;
+
+    --ace-background: #f6f8fa;
+    --ace-string: #032f62;
+    --ace-numeric: #005cc5;
+    --ace-function: #6f42c1;
+    --ace-comment: #9299a2;
+    --ace-line: #8b0000;
+    --ace-variable: #000;
+
+    --dropdown-title-color: #c9d1d9;
+    --dropdown-title-background: #0d1117;
+    --dropdown-list-color: #fff;
+    --dropdown-list-background: #0d1117;
+    --dropdown-border: #a5a5a5;
+    --dropdown-hover-background: #375eab;
+    --dropdown-selected-background: #1f3663;
+
+    --footer-color: #646262;
+}
+
+@media (prefers-color-scheme: dark){
+    :root {
+        --toolbar: #000;
```

Comment:
this is not used anywhere, did you mean `--toolbar-color` ?

Reply:
Oh yes, my mistake 👋🏻